### PR TITLE
Shortform array

### PR DESCRIPTION
**Pull request by bbannon**
The rest of the shortform-array conversions. 
Yes, it's 94 files. It was either this or 94 branches and 94 pull requests.
I think it would take me six months to do it that way and you to merge it in.
I'm hoping to not have as much time to work on WebCalendar in six months.

I left some notes as // Craig
for you, or me, or someone to work on. Eventually.

### CODE REVIEW

**Comment by craigk5n** [comment]
I think it is pretty standard practice to bunch changes related to the same issue within a single pull request.

**Comment by craigk5n** [comment]
Seems like there are some PHP syntax errors.  When I run 'php -l' on all the PHP files, I get the following errors:

Parse error: parse error, expecting `']'' in ./admin.php on line 294
Errors parsing ./admin.php

Parse error: parse error in ./includes/index.php on line 1
Errors parsing ./includes/index.php

Parse error: parse error in ./includes/user-app-postnuke.php on line 364
Errors parsing ./includes/user-app-postnuke.php

Parse error: parse error in ./layers_ajax.php on line 59
Errors parsing ./layers_ajax.php

Parse error: parse error, expecting `']'' in ./themes/default_admin.php on line 137
Errors parsing ./themes/default_admin.php

**Comment by bbannon** [comment]
(sigh) I very carefully checked all the []. I seem to have missed some extra ")"s in /includes/user-app-postnuke.php. And missed a closing single quote in /themes/default_admin.php.
If I fix them, re add/commit/push, will the pull request still work? Or do I need to re do that?

**Comment by craigk5n** [comment]
FYI, here's the link to my commit: https://github.com/craigk5n/webcalendar/pull/26/commits/c8bca24fe8b62fd4f549a91ba5f8456f6d1bd456

**Comment by craigk5n** [comment]
Once all your code compiles without errors (please use "php -l" on each php file), you need to commit the changes and then push the changes up to your remote branch.  The changes should automatically appear on this pull request.

File: layers_ajax.php [hunk 1]
```diff
@@ -72,6 +72,11 @@
       'source' => $layer['cal_layeruser'],
       'color' => $layer['cal_color'],
       'dups' => $layer['cal_dups'],
+// Craig. This looks like a bug. I can't find $layerfullname set anywhere.
+// And the only table with a fullname is webcal_entry_ext_user, which doesn't seem to be called here.
+// Even $fullname doesn't seem to fit.
+// Is it supposed to be maybe $layer['cal_login'] or $layer['cal_layeruser']?
+
       'fullname' => $layerfullname];
```
Comment (craigk5n) [inline layers_ajax.php]: The fullname was supposed to be a convenience for the calling pages.  Not sure if we are still using it anywhere though.  We used to have repeated sections of code that would assign fullname based on first, last and username (mostly handling empty values).

Comment (bbannon) [inline layers_ajax.php]: Well, since it's not set anywhere I can find, I suggest this line be changed to
'fullname' => layer['cal_layeruser'];

**Comment by craigk5n** [comment]
Hey, @bbannon .  This is now merged.  I did have to fix one PHP compile error beforehand.

File: layers_ajax.php [hunk 2]
```diff
@@ -65,14 +65,19 @@
...
       'color' => $layer['cal_color'],
       'dups' => $layer['cal_dups'],
-      'fullname' => $layerfullname );
+// Craig. This looks like a bug. I can't find $layerfullname set anywhere.
+// And the only table with a fullname is webcal_entry_ext_user, which doesn't seem to be called here.
+// Even $fullname doesn't seem to fit.
+// Is it supposed to be maybe $layer['cal_login'] or $layer['cal_layeruser']?
+
+      'fullname' => $layerfullname];
   }
```
Comment (craigk5n) [inline layers_ajax.php]: The call to user_load_variables with 'layer' as the prefix parameter will cause that method to create a variable named "layer" . "fullname".  So, this code is correct.

You can try the ajax request directly in your browser to see that fullname does get populated in the JSON returned:

http://yourserver/webcalendar/layers_ajax.php?action=list

Here is what I get back:

`{"error":0,"status":"OK","message":"","layers":[{"id":1,"source":"jsmith","color":"#000000","dups":"N","fullname":"Joe Smith"}]}`

**Pull request merged**